tests/unittests/operators/test_logging_utils.py

```
@@ -58,6 +65,38 @@ def test_progress_bar_resets_on_new_bar(self, tee, sink):
         pcts = [int(line.split("%")[0].strip()) for line in lines]
         assert pcts == [90, 91, 0, 1]
 
+    def test_barless_progress_within_same_percent_is_throttled(
+        self, tee, sink
+    ):
+        for n in range(15488, 16064, 64):
+            tee.write(_eta_barless(1, n))
+        tee.drain()
+
+        lines = [l for l in sink.getvalue().splitlines() if l.strip()]
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)